velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -339,6 +350,36 @@ TEST_P(PrestoSerializerTest, dictionaryWithExtraNulls) {
   testRoundTrip(dictionary);
 }
 
+// TODO: re-enable after we support dictionary encoding in Presto serializer.
+TEST_P(PrestoSerializerTest, DISABLED_dictionaryEncoding) {
```

Comment:
I'm not sure I see value in adding a disabled test. Can we remove it from this PR?